src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -9129,6 +9129,66 @@ public void ReportAnalyzerOutput()
             CleanupAllGeneratedFiles(srcFile.Path);
         }
 
+        [Fact]
+        [WorkItem(40926, "https://github.com/dotnet/roslyn/issues/40926")]
+        public void SkipAnalyzersParse()
```

Comment:
Feel like there are two tests missing:

1. Verify that even with `/skipAnalyzers-` the assembly is still loaded into the process. 
1. Verify that when a DLL has both an analyzer and a source generator that we load the source generator when `/skipAnalyzers-` is done. 

Honestly the second can really be a good proxy for the first.

Reply:
Sure, I will add both tests.

Reply:
Added a new test `TestSourceGeneratorsWithAnalyzers` which should test all combinations.